src/game/gameOverModal.scss

```diff
@@ -0,0 +1,58 @@
+.modal {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  color: white;
+  &::before {
+    position: absolute;
+    content: "";
+    top: 0;
+    left: 15px;
+    right: 15px;
+    bottom: 0;
+    background-color: rgba(0, 0, 0, 0.8);
+    @media screen and (min-width: $medium-breakpoint) {
+      left: 45px;
+      right: 45px;
+    }
+  }
+  &__content {
+    max-width: 80vw;
+    max-height: 80vh;
+    z-index: 999;
+    h2 {
+      border-bottom: 5px solid white;
+      line-height: 1.4;
+      font-size: $l-font-size-mobile;
+      margin: 0;
+      @media screen and(min-width: $medium-breakpoint) {
+        font-size: $l-font-size-desktop;
+      }
+    }
+    h3 {
+      font-size: $xs-font-size-mobile;
+      @media screen and(min-width: $medium-breakpoint) {
+        font-size: $xs-font-size-desktop;
+      }
+    }
+  }
+  &__actions {
```

Comment:
The space between `h3` and `modal-actions` is 99 px on design for desktop. Let's round it and make it 100 px.

Reply:
Overwritten to 100px